packages/agreement-process/src/services/readmodel/agreementQuery.ts

```diff
@@ -18,6 +19,18 @@ export function agreementQueryBuilder(readModelService: ReadModelService) {
       offset: number
     ): Promise<ListResult<Agreement>> =>
       readModelService.getAgreements(filters, limit, offset),
+    listConsumers: (
```

Comment:
@MalpenZibo Another change involving your PR, discussed with @Viktor-K yesterday as the one in the other comment: we moved `listConsumers` into `agreementQuery.ts` because it performs queries on the `agreements` collection, and it would also have been redundant to have another `producersQuery.ts` file. Same as above, you could port this change in #125 then I rebase